src/service/comparison_vs_peers/comparison_vs_peers_service.py

```diff
@@ -94,11 +97,17 @@ def get_comparison_vs_data(self, data: dict, access: bool) -> list:
 
         for id in data:
             company_data = data[id]
+            company_revenue = company_data.get("actuals_revenue")
+            rule_of_40_revenue = (
+                int(company_revenue)
```

Comment:
Is it needed to cast to int the revenue?
Can we use calculate_base_metric from calculator here?